source/draw/DrawState.ooc

```diff
@@ -36,6 +36,7 @@ DrawState: cover {
 	_destinationNormalized := FloatBox2D new(0.0f, 0.0f, 1.0f, 1.0f)
 	_sourceNormalized := FloatBox2D new(0.0f, 0.0f, 1.0f, 1.0f)
 	_focalLengthNormalized := 0.0f // Relative to image width
+	flatTransform := FloatTransform2D identity // 2D scaling done after projection matrix but before w division
```

Comment:
Setting different variables with the same name leads to confusion since we might use both at the same time and then someone erases the first assignment believing that it is dead code.